packages/mediacenter/kodi/scripts/pastekodi

```diff
@@ -65,17 +77,15 @@ fi
   fi
   if [ "${LIBREELEC_PROJECT}" = "RPi" ]; then
     echo "RPi Hardware Revision: $(vcgencmd otp_dump | grep 30: | cut -d: -f2)"
+    echo "RPi $(vcgencmd get_throttled)"
   fi
 
   cat_file "${LOG_FILE}"
 
-  journalctl -a | cat_data "journalctl -a"
+  journalctl -a -o short-precise | cat_data "journalctl -a"
```

Comment:
Another thought: using `debugging` boot parameter result in creating a persistent journal. Only paste the logs we like to see in this case. Having an empty previous journal by default does not harm.

```suggestion
  journalctl -a -o short-precise -b -0 | cat_data "journalctl -a (current boot)"
  journalctl -a -o short-precise -b -1 | cat_data "journalctl -a (previous boot)"
```

Reply:
I'd rather leave this as is, if we add too much info we risk going over the pastesite limit (IIRC 1MB or so) and pastes get rejected.

Also usage "debugging" boot parameter is rather niche and results in too much logspam (ISTR connman etc get very chatty with it.